docs/guides/convert-to-dictionary.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Convert to Dictionary
+---
+
+# Convert to Dictionary
+
+Strawberry provides a utility function to convert a Strawberry object to a
+dictionary.
+
+You can use `strawberry.asdict(...)` function:
+
+```python
+@strawberry.type
+class User:
+    name: str
+    age: int
+
+
+# should be {"name": "Lorem", "age": 25}
+user_dict = strawberry.asdict(User(name="Lorem", age=25))
+```
+
+> Note: This function is not recursive, so if you have nested objects, you will
+> need to convert them manually.
```

Comment:
My understanding is that `dataclasses.asdict` **is** recursive. Is that not the case? 

This test seems to confirm that as well: https://github.com/strawberry-graphql/strawberry/pull/2417/files#diff-87d39535d5c687a3849638b7599cc65e1baedd3e457fab99f32dc37630e8ee2bR22-R48

Reply:
you're right! https://github.com/python/cpython/blob/5d868efde9ea08b39357b962a02a0294aa97689c/Lib/dataclasses.py#L1294-L1301

I'll remove this line